sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/AvroUtils.java

```diff
@@ -765,8 +765,8 @@ private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundE
       if (logicalType instanceof LogicalTypes.Decimal) {
         fieldType = FieldType.DECIMAL;
       } else if (logicalType instanceof LogicalTypes.TimestampMillis) {
-        // TODO: There is a desire to move Beam schema DATETIME to a micros representation. When
```

Comment:
This is unrelated to the representation of Beam schema `DATETIME`. The correct TODO comment for this is here: https://github.com/apache/beam/blob/961cf0936d7065bf6089e666fbbc6b83da266df2/sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/AvroUtils.java#L883-L884